### PR TITLE
Updated treemap

### PR DESCRIPTION
**Pull request by russ430**
Made the treemap responsive to the viewport (currently the width and height are set to 70vw and 50vh respectively). These values can be changed in the 'Container' component created at the top of the Treemap file. 

I also changed the colors to a blue gradient that maps to the values of each cell. The function to create the colors separates the data into 3 even ranges which will correspond to a specific color (darker = higher value, lighter = lower).

These changes are in response to comments by @alexjball on the [original pull request](https://github.com/codeforboston/jobhopper/pull/125#pullrequestreview-508545710)

One of the difficulties I'm having is in the range call for the scaleQuantile function on line 81 in the Treemap.tsx file Typescript is complaining that the values must be number values. However in the [documentation](https://github.com/d3/d3-scale#quantile_range) it says elements in the range can be any value. Not sure how to resolve the issue in Typescript, any help on that would be great!

I have a preview of what the treemap looks like as well below.

blue gradient treemap

### CODE REVIEW

**Comment by jedpittman** [comment]
@russ430 , this looks like a good start. 

Can you please work with @alexjball and ensure that we create issues for the items from the code review that we're going to implement? Also, if possible, let's make these issues small and targetted. There are lots of changes in this request and it is not clear which ones map to which desired effect. Keeping the issues small makes that easier to track.

In general, the flow should be: create the issue, discuss together Tuesday, schedule into a sprint, then do the work. If you're doing work that is "off sprint" then we might miss items that we had promised. Also it might waste time.

**Comment by alexjball** [comment]
Hey @jedpittman,

These are all responses to #125, which got merged before we had time to address them. They are all relevant to the original issue #81

**Comment by edward-malouf** [comment]
@russ430 This is good progress! One note; the transition share must be expressed in percentages, not decimals. For instance, Secretaries and administrative assistants... has a .0391 transition share. It needs to be expressed as 3.9%. This may be a refinement for later, just thought I would flag it now.

File: frontend/src/ui/D3Visualizations/Treemap.tsx [hunk 1]
```diff
@@ -54,43 +74,83 @@ export default function Treemap({ height, width, data }: TreemapProps) {
...
       .attr('transform', d => `translate(${d.x0},${d.y0})`);
 
-    // lighten colors
-    const fader = (color: string): string =>
-      d3.interpolateRgb(color, white)(0.3);
-
     // select color within d3.schemeSet3 (contains array of 12 colors)
-    const colorScale = d3.scaleOrdinal(d3.schemeSet3.map(fader));
+    const colorScale = d3
+      .scaleQuantile()
```
Comment (alexjball) [inline frontend/src/ui/D3Visualizations/Treemap.tsx:79]: scaleQuantile is a generic function with a type parameter for the type of the range. It defaults to `number`, which is why you're getting the typescript error. I guess the compiler isn't smart enough to infer the value. Anyway, I am able to fix the error by replacing this line with

`.scaleQuantile<string>`

**Review by alexjball — APPROVED** [review]

**Comment by russ430** [comment]
@jedpittman this is ready for merge

**Comment by jedpittman** [comment]
merging now that I've seen Alex R's comment.

**Comment by jedpittman** [comment]
merging these additional changes related to #125 and #81

**Pull request merged**